src/SelfAssessment/index.js

```diff
@@ -76,6 +76,7 @@ const Assessment = ({ navigation }) => {
     // to support dynamic questions
 
     // memoize assessment question
+    /* eslint-disable react/display-name */
```

Comment:
I believe this comment could be avoided if we do not use an anonymous function, perhaps doing:
```
function assesmentQuestionMemoized({ navigation, route }) {
  return {...}
}
```
Or extracting the function into a named component might fix this?

Reply:
this file go removed anyways, so no longer an issue!